changeStreams.js

```diff
@@ -0,0 +1,100 @@
+const { MongoClient } = require('mongodb');
+
+async function main() {
+    /**
+     * Connection URI. Update <username>, <password>, and <your-cluster-url> to reflect your cluster.
+     * See http://bit.ly/NodeDocs_lauren for more details
+     */
+    const uri = "mongodb+srv://<username>:<password>@<your-cluster-url>/test?retryWrites=true&w=majority";
+
+    /**
+     * The Mongo Client you will use to interact with your database
+     * See bit.ly/Node_MongoClient for more details
+     */
+    const client = new MongoClient(uri);
+
+    try {
+        // Connect to the MongoDB cluster
+        await client.connect();
+
+        // Make the appropriate DB calls
+        
+        /**
+         * An aggregation pipeline that matches on new listings in the country of Australia and the Sydney market
+         */
+        const pipeline = [
+            {
+                '$match': {
+                    'operationType': 'insert',
+                    'fullDocument.address.country': 'Australia',
+                    'fullDocument.address.market': 'Sydney'
+                }
+            }
+        ];
+        
+        // This script contains two ways to monitor new listings in the listingsAndReviews collection.
```

Comment:
instead of a custom method `monitorListingsUsingOn`, I suggest rewriting this like so:

```
const collection = client.db("sample_airbnb").collection("listingsAndReviews");
const changeStream = collection.watch(pipeline);
changeStream.on('change', (next) => console.log(next));

await wait(30000); // 30s
```

and introduce a `wait` helper method:
```
const wait = ms => new Promise(resolve => setTimeout(resolve, ms));
```